atd-etl/prefect/afd_ems_incidents/afd_incident_upload.py

```diff
@@ -0,0 +1,279 @@
+#!/usr/bin/env python
+
+"""
+Name: AFD Incident Uploads
+Description: This flow uploads AFD Incident Response CSVs (AFD Contact: Gus). 
+    The data is emailed to atd-afd-incident-data@austinmobility.io daily ~ 6:30AM. From there it
+    gets forwarded to a S3 bucket via AWS Simple Email Service.
+"""
+
+import prefect
+from prefect import Flow, task, Parameter, case
+import datetime
+import json
+import boto3
+import pprint
+import os
+import time
+import email
+import tempfile
+import shutil
+import pandas
+from datetime import datetime, timedelta
+import psycopg2
+import psycopg2.extras
+from prefect.backend import get_key_value
+
+pp = pprint.PrettyPrinter(indent=4)
+
+kv_data = get_key_value(key="Vision Zero Development")
+environment_variables = json.loads(kv_data)
+
+DB_USERNAME = None
+DB_PASSWORD = None
+DB_HOSTNAME = None
+DB_PORT = None
+DB_DATABASE = None
+AWS_ACCESS_KEY_ID = None
+AWS_SECRET_ACCESS_KEY = None
+AFD_S3_SOURCE_BUCKET = None
+AFD_S3_ARCHIVE_BUCKET = None
+AFD_S3_SOURCE_PREFIX = None
+AFD_S3_ARCHIVE_PREFIX = None
+
+# Retrieve the db configuration
+if False:
```

Comment:
I'm confused by this, why if False?

Reply:
oh, is this so someone can toggle between getting the config out of the kv store instead of local environment?